### PR TITLE
Add BigQuery Connector Example

### PR DESCRIPTION
**Pull request by KasunAratthanage**

### CODE REVIEW

File: en/micro-integrator/docs/references/connectors/bigquery-connector/bigquery-connector-example.md [hunk 1]
```diff
@@ -0,0 +1,722 @@
+# BigQuery Connector Example
+
+The BigQuery connector allows you to access the [BigQuery REST API](https://cloud.google.com/bigquery/docs/reference/rest) through WSO2 ESB. BigQuery is a tool that allows you to execute SQL-like queries on large amounts of data at outstanding speeds. It is a serverless Software as a Service that supports querying using ANSI SQL.
```
Comment (samgnaniah) [inline en/micro-integrator/docs/references/connectors/bigquery-connector/bigquery-connector-example.md:3]:
```suggestion
The BigQuery connector allows you to access the [BigQuery REST API](https://cloud.google.com/bigquery/docs/reference/rest) through WSO2 EI. BigQuery is a tool that allows you to execute SQL-like queries on large amounts of data at outstanding speeds. It is a serverless Software as a Service that supports querying using ANSI SQL.
```

File: en/micro-integrator/docs/references/connectors/bigquery-connector/bigquery-connector-example.md [hunk 2]
```diff
@@ -0,0 +1,722 @@
...
+
+* `/gettabledetails`: This is used to get get created table details from the BigQuery table by ID.
+* `/insertdetails` : This is used to inserts the data into the table.
+* `/getdetails` : This is used to retrieves table data from a specified set of rows.
+* `/runQuery` : The runQuery operation runs an SQL query (BigQuery) and returns results if the query completes within a specified timeout.              
+
+ > **Note**: Before start this scenario user need to create a **project** in the BigQuery. As next step need to create a **Dataset** and under that Dataset you have to have **Table**. For more information about these operations, please refer to the [Setting up the BigQuery Environment](bigquery-connector-configuration.md). 
+
+The following diagram shows the overall solution. User can invoke the table schema level details from the `gettabledetails` resource. Using this response details, the API caller can insert data in to the created table. If user need to retrieve table data from a specified set of rows, need to invoke `getdetails` resource. Finally `/runQuery` resource runs an SQL query (BigQuery) and returns results back to the API caller.
```
Comment (samgnaniah) [inline en/micro-integrator/docs/references/connectors/bigquery-connector/bigquery-connector-example.md:23]:
```suggestion
The following diagram shows the overall solution. User can invoke the table schema level details from the `gettabledetails` resource. Using the response details, the API caller can insert data into the created table. If users need to retrieve table data from a specified set of rows, they need to invoke the `getdetails` resource. Finally `/runQuery` resource runs an SQL query (BigQuery) and returns results back to the API caller.
```

File: en/micro-integrator/docs/references/connectors/bigquery-connector/bigquery-connector-example.md [hunk 3]
```diff
@@ -0,0 +1,722 @@
...
+
+stores WSO2 stock volume details into the list, and then receives it back and removes unwanted hash map items. To invoke each operation, the user uses the same API.
```
Comment (samgnaniah) [inline en/micro-integrator/docs/references/connectors/bigquery-connector/bigquery-connector-example.md:25]: The beginning of this seems missing.

File: en/micro-integrator/docs/references/connectors/bigquery-connector/bigquery-connector-example.md [hunk 4]
```diff
@@ -0,0 +1,722 @@
...
+4. Run an SQL query (BigQuery) and retrieve details from BigQuery table.
+
+All four operations are exposed via an `bigquery-testAPI` API. The API with the context `/resources` has four resources.
+
+* `/gettabledetails`: This is used to get get created table details from the BigQuery table by ID.
+* `/insertdetails` : This is used to inserts the data into the table.
+* `/getdetails` : This is used to retrieves table data from a specified set of rows.
+* `/runQuery` : The runQuery operation runs an SQL query (BigQuery) and returns results if the query completes within a specified timeout.              
+
+ > **Note**: Before start this scenario user need to create a **project** in the BigQuery. As next step need to create a **Dataset** and under that Dataset you have to have **Table**. For more information about these operations, please refer to the [Setting up the BigQuery Environment](bigquery-connector-configuration.md).
```
Comment (samgnaniah) [inline en/micro-integrator/docs/references/connectors/bigquery-connector/bigquery-connector-example.md:21]:
```suggestion
 > **Note**: Before starting this scenario, you need to create a **project** in BigQuery. Next, create a **Dataset** and under that Dataset you have to have **Table**. For more information about these operations, please refer to the [Setting up the BigQuery Environment](bigquery-connector-configuration.md). 
```

File: en/micro-integrator/docs/references/connectors/bigquery-connector/bigquery-connector-example.md (hunk 1 above)
Comment (KasunAratthanage) [reply en/micro-integrator/docs/references/connectors/bigquery-connector/bigquery-connector-example.md:3]: Updated.

File: en/micro-integrator/docs/references/connectors/bigquery-connector/bigquery-connector-example.md (hunk 2 above)
Comment (KasunAratthanage) [reply en/micro-integrator/docs/references/connectors/bigquery-connector/bigquery-connector-example.md:23]: Updated.

File: en/micro-integrator/docs/references/connectors/bigquery-connector/bigquery-connector-example.md (hunk 4 above)
Comment (KasunAratthanage) [reply en/micro-integrator/docs/references/connectors/bigquery-connector/bigquery-connector-example.md:21]: Updated.

File: en/micro-integrator/docs/references/connectors/bigquery-connector/bigquery-connector-example.md (hunk 3 above)
Comment (KasunAratthanage) [reply en/micro-integrator/docs/references/connectors/bigquery-connector/bigquery-connector-example.md:25]: Updated.

File: en/micro-integrator/docs/references/connectors/bigquery-connector/bigquery-connector-example.md [hunk 5]
```diff
@@ -0,0 +1,720 @@
+# BigQuery Connector Example
+
+The BigQuery connector allows you to access the [BigQuery REST API](https://cloud.google.com/bigquery/docs/reference/rest) through WSO2 EI. BigQuery is a tool that allows you to execute SQL-like queries on large amounts of data at outstanding speeds. It is a serverless Software as a Service that supports querying using ANSI SQL.
+
```
Comment (abeykoon) [inline en/micro-integrator/docs/references/connectors/bigquery-connector/bigquery-connector-example.md:4]: Currently it supports only read only operations. It is assumed a third party system is using BigQuery and the Big query connector is used to query information.

Comment (abeykoon) [reply en/micro-integrator/docs/references/connectors/bigquery-connector/bigquery-connector-example.md:4]: if true, better mention it ahead.

File: en/micro-integrator/docs/references/connectors/bigquery-connector/bigquery-connector-example.md [hunk 6]
```diff
@@ -0,0 +1,720 @@
...
+Given below is a sample scenario that demonstrates how to work with the WSO2 EI BigQuery Connector:
+
+1. The user sends the request to invoke an API to get created table details from the BigQuery. This REST call will retrieve schema level information and send it back to the API caller.
+2. Insert data in to the created table.
+3. Retrieve inserted details from the BigQuery table.
+4. Run an SQL query (BigQuery) and retrieve details from BigQuery table.
+
+All four operations are exposed via an `bigquery-testAPI` API. The API with the context `/resources` has four resources.
+
+* `/gettabledetails`: This is used to get get created table details from the BigQuery table by ID.
```
Comment (abeykoon) [inline en/micro-integrator/docs/references/connectors/bigquery-connector/bigquery-connector-example.md:16]: table details - shall we say what they are?

**Pull request merged**